build/grafana/dashboard-allocations.yaml

```diff
@@ -170,7 +170,6 @@ data:
           "nullPointMode": "connected",
           "nullText": null,
           "options": {},
-          "pluginVersion": "6.2.4",
```

Comment:
Just for my learning purpose, why these `pluginVersion` are not needed?

Reply:
It's also a reference to the Grafana version. Leaving it out here so that it will use the most recent stable version instead of a specific version.